File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
from collections import Counter
import heapq
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        
        frequency_array = Counter(nums)
        print(frequency_array)
        
        arr = []
        for num, count in frequency_array.items():
            arr.append([count, num])
        arr.sort()

        print(arr)

        res = []
        while len(res) < k:
            res.append(arr.pop()[1])
        return res

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int n = nums.size();

        int counter = 0;
        while (counter < n) {
            int val = nums[counter];
            for (int i = 0; i < n; i++) {
                if (nums[i] == val && i != counter) {
                    return true;
                }
            }

            counter++;
        }

        return false;
        
    }
};
